pylink/jlink.py

```diff
@@ -4719,3 +4719,148 @@ def swo_read_stimulus(self, port, num_bytes):
         bytes_read = self._dll.JLINKARM_SWO_ReadStimulus(port, buf, buf_size)
 
         return list(buf)[:bytes_read]
+
+###############################################################################
+#
+# Real Time Terminal (RTT) API
+#
+###############################################################################
+
+    @open_required
+    def rtt_start(self):
+        """Starts RTT processing, including background read of target data.
+        Args:
+          self (JLink): the ``JLink`` instance
+
+        Raises:
+          JLinkRTTException if the underlying JLINK_RTTERMINAL_Control call fails.
+        """
+
```

Comment:
NIT: no newline between the closing docstring and the first line of code.

Reply:
Done (here and everywhere else)